packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -0,0 +1,459 @@
+  <h1 class="capitalize text-lg font-bold py-8 px-8">Registries</h1>
+  <div class="container mx-auto">
+    <!-- Registries table start -->
+    <div class="w-full border-t border-b border-gray-600">
+      <div class="flex w-full">
+        <div class="flex-1 text-left py-4 pl-10 text-sm font-bold w-auto">Registry Location</div>
+        <div class="text-left py-4 text-sm font-bold w-1/4">Username</div>
+        <div class="text-left py-4 text-sm font-bold w-2/5">Password</div>
+      </div>
+
+      {#each registryStore as registry}
+        <!-- Registry row start -->
+        <div class="flex flex-col w-full border-t border-gray-600">
+          <div class="flex flex-row">
+            <!-- Server URL -->
+            <div class="flex-1 pt-5 pb-5 pl-12 text-sm w-auto">{registry.serverUrl}</div>
+
+            <!-- Username -->
+            <div class="pt-5 pb-5 text-sm w-1/4">
+              {#if isRegistryDirty(registry)}
+                <div class="text-left h-7 pr-5 -mt-0.5 -mb-2 text-sm w-full">
+                  <input
+                    type="text"
+                    placeholder="Username"
+                    value="{getDirtyRegistry(registry).username}"
+                    on:input="{event => updateRegistryUsername(event, registry)}"
+                    class="block w-full h-full px-3 transition ease-in-out delay-50 text-gray-400 placeholder-gray-400 placeholder:italic placeholder:opacity-40 rounded-sm focus:outline-none focus:ring-1 focus:ring-gray-200 focus:ring-opacity-20" />
+                </div>
+              {:else if isDumbRegistry(registry)}
+                <button class="font-bold" on:click="{() => markRegistryAsDirty(registry)}">Login now</button>
+              {:else}
+                {registry.username}
+              {/if}
+            </div>
+
+            <!-- Password -->
+            <div class="pt-5 pb-5 text-sm w-full w-2/5">
+              <div class="flex flex-row">
+                {#if isRegistryDirty(registry)}
+                  <div class="text-left h-7 pr-5 -mt-0.5 -mb-2 text-sm w-full">
+                    <input
+                      type="text"
+                      placeholder="Password"
+                      value="{getDirtyRegistry(registry).secret}"
+                      on:input="{event => updateRegistryPassword(event, registry)}"
+                      class="block w-full h-full px-3 transition ease-in-out delay-50 text-gray-400 placeholder-gray-400 placeholder:italic placeholder:opacity-40 rounded-sm focus:outline-none focus:ring-1 focus:ring-gray-200 focus:ring-opacity-20" />
+                  </div>
+                  <div class="h-7 pr-5 -mt-0.5 -mb-2 text-sm w-28">
+                    <button
+                      on:click="{() => loginToRegistry(registry)}"
+                      class="bg-[#6e4ff5] hover:bg-[#613ff4] transition ease-in-out delay-50 hover:cursor-pointer disabled:bg-gray-400 w-full h-full rounded-md shadow hover:shadow-lg"
```

Comment:
I think we should not introduce new colors, just use bg-zinc-* and then we could change the value of the palette definition

Reply:
removed and used predefined colours